lib/src/space/grounding.rs

```diff
@@ -47,7 +50,25 @@ impl<'a> Iterator for GroundingSpaceIter<'a> {
     }
 }
 
-fn atom_to_trie_key(atom: &Atom) -> TrieKey<SymbolAtom> {
+impl Serializer for DefaultHasher { // there are much speedier hashers, but I didn't want to introduce a dependency
```

Comment:
I think these impls probably belong in `serial.rs` instead of `grounding.rs` since they're not specific to a GroundingSpace.

Also minor style point, but I usually reserve var names beginning with '_' for unused vars.  Unclear what it signifies here.